lib/pbench/cli/server/shell.py

```diff
@@ -194,6 +195,26 @@ def main():
         logger.info("Created database {}", db_uri)
     Database.init_db(server_config, logger)
 
+    # Create the crontab file from the server configuration.
+    os.environ["PATH"] = ":".join([server_config.BINDIR, os.environ["PATH"]])
+    cp = subprocess.run(
+        ["pbench-create-crontab", crontab_dir],
+        cwd=server_config.log_dir,
+    )
+    if cp.returncode != 0:
+        logger.error(
+            "Failed to create crontab file from configuration: {}", cp.returncode
+        )
+        sys.exit(1)
```

Comment:
Do we really want to repeat this each time the server is started? In theory, it means we can dynamically change the config file tasks, but that's unlikely to be useful. Seems this could be gated on whether the `crontab` file exists already.

Reply:
Given that server start events are likely to be relatively rare (outside of active development...), the effort to be saved by not re-creating the crontab file seems very small, whereas, as you pointed out, unconditionally recreating it does offer a certain feature (not to mention keeping the code simpler).